pkg/judge/rego.go

```diff
@@ -19,17 +24,24 @@ type RegoJudge struct {
 type RegoOpts struct {
 }
 
-func NewRegoJudge(opts *RegoOpts) (*RegoJudge, error) {
+func NewRegoJudge(opts *RegoOpts, additionalResourcesStr []string) (*RegoJudge, error) {
 	ctx := context.Background()
 
+	var additionalKinds []schema.GroupVersionKind
+	for _, ar := range additionalResourcesStr {
+		gvr, _ := schema.ParseKindArg(ar)
+		additionalKinds = append(additionalKinds, *gvr)
+	}
+
 	r := rego.New(
 		rego.Query("data[_].main"),
 	)
 
 	statikFS, err := fs.New()
 
-	fs.Walk(statikFS, "/",
+	err = fs.Walk(statikFS, "/",
 		func(path string, info os.FileInfo, err error) error {
```

Comment:
this is too large now, please make it a separate go function

Reply:
This should be addressed now.